src/Compute/Compute/Generated/Image/ImageCreateOrUpdateMethod.cs

```diff
@@ -82,7 +83,7 @@ public override void ExecuteCmdlet()
         public SwitchParameter AsJob { get; set; }
     }
 
-    [Cmdlet(VerbsData.Update, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Image", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [Cmdlet(VerbsData.Update, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Image", DefaultParameterSetName = "ObjectParameter", SupportsShouldProcess = true)]
```

Comment:
DefaultParameter should always be the default parameter set - it doesn't make sense to prompt for an object instead of a string if no parameters are provided.

Reply:
@maddieclayton  Since I changed Image from mandatory to optional for DefaultParameter set, if I set "DefaultParameter" set to default, then Update-AzImage with -Image object throw an error because it also looks for resource group name and image name.  So, I changed default parameter set to "ObjectParameter" set until we can remove Image parameter from "DefaultParameter" set in the upcoming breaking change release.

Reply:
This is fine for now then